lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,45 @@ def approved?
           end
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
+          options[:destination_branch] ||= 'master'
+
+          message = "#{options[:message]}"
+
+          if "#{options[:pull_request_number]}".length > 0
+            message << "\nMerges ##{options[:pull_request_number]}\n"
+          end
+
+          if lgtm_authors = Array(options[:lgtm_authors]) and lgtm_authors.any?
+            message << "\nLGTM given by: @#{lgtm_authors.join(', @')}\n"
+          end
+
+          data = {
+            "commit_title" => options[:title],
+            "commit_message" => options[:message],
+            "sha" => options[:head].sha,
+            "squash" => true
+          }
+          
+          res = GitReflow::GitServer::GitHub.connection.pull_requests.merge(
```

Comment:
We should be clear with our variable names.  Let's change `res` to `merge_response`